pkg/session/session.go

```diff
@@ -3221,12 +3221,7 @@ func InitDDLJobTables(store kv.Storage, targetVer meta.DDLTableVersion) error {
 		targetTables = BackfillTables
 	}
 	return kv.RunInNewTxn(kv.WithInternalSourceType(context.Background(), kv.InternalTxnDDL), store, true, func(_ context.Context, txn kv.Transaction) error {
-		var t *meta.Meta
-		if variable.EnableFastCreateTable.Load() {
-			t = meta.NewMeta(txn, meta.WithUpdateTableName())
-		} else {
```

Comment:
Is there a plan to clean these `tableName -> tableID` kvs on old cluster?

Reply:
fast create is expeironmental, we will write in doc to let customer disable it before upgrade

Reply:
https://github.com/pingcap/docs-cn/pull/18214 doc pr